apps/ios/SimpleX SE/ShareView.swift

```diff
@@ -51,27 +52,22 @@ struct ShareView: View {
             text: $model.search,
             placement: .navigationBarDrawer(displayMode: .always)
         )
-        .alert($model.errorAlert) { _ in
-            Button("Ok") { model.completion() }
+        .alert($model.errorAlert) { alert in
+            if alert.title == "Network" {
```

Comment:
maybe better to check it has no actions than for specific title?

Reply:
Wait - this is actually not needed anymore. Removed.